src/Typesense/Documents.js

```diff
@@ -76,7 +76,9 @@ export default class Documents {
     return this._apiCall.get(this._endpointPath('export'), options)
   }
 
-  search (searchParameters, {cacheSearchResultsForSeconds = this._configuration.cacheSearchResultsForSeconds} = {}) {
+  search (searchParameters,
+    {cacheSearchResultsForSeconds = this._configuration.cacheSearchResultsForSeconds} = {},
+    {abortSignal = null} = {}) {
```

Comment:
@okhobb Sorry I just noticed this, but any reason `abortSignal` is a separate parameter in the method call, vs being part of the previous object? 

For eg: can the definition be:

```
search (
  searchParameters, 
  {
    cacheSearchResultsForSeconds = this._configuration.cacheSearchResultsForSeconds,
    abortSignal = null
  } = {}
)
```

Reply:
No good reason other than avoiding conflicts/not knowing the codebase well. Moved it forward in the recent push.